person.rb

```diff
@@ -6,14 +6,15 @@ def correct_name
 
 class Person < Nameable
   attr_accessor :name, :age
-  attr_reader :id
+  attr_reader :id, :rentals
 
   def initialize(age, name = 'Unknown', parent_permission: true)
     @id = generate_id
     @name = name
     @age = age
     @parent_permission = parent_permission
     super()
+    @rentals = []
   end
 
```

Comment:
- After adding the books, we also need to add the method for the rentals for every book rented by the person since the rentals in an intermediary between a person and a book to establish the has-many relationship.

```ruby
def add_rental(book, date)
    Rental.new(date, self, book)
  end
```
- The date is included to keep track of the date each book is rented and also, the person itself should be sent as a parameter to create the new rental achieving the 'has-many' association` which is why the self is added in the parameter.

Reply:
Done!

Reply:
Addressed